### PR TITLE
Answer confirmation & beautification

### PR DESCRIPTION
**Pull request by DoruM**
Added multiple changes:
- dialog box with ability to reject or add a call
- added overlay describing the name of the person we are talking with
- showing and hiding "Hang Up" and "Call" buttons appropriately
- enhanced logging

### CODE REVIEW

File: gateway/webrtc/client.js [hunk 1]
```diff
@@ -216,10 +221,16 @@ loginBtn.addEventListener("click", function(event) {
     consumer.setMessageListener(function(message) {
         handleMessage(message);
     });
-
+    console.log("Exiting loginBtn.click");
 });
+// sleep time expects milliseconds
```
Comment (dpwspoon) [inline gateway/webrtc/client.js]: Do we need this commented out code?

File: gateway/webrtc/client.js [hunk 2]
```diff
@@ -245,31 +256,86 @@ function handleVideo(myStream) {
...
+        }
+        console.log("Exiting onaddstream");
     };
 
     // Setup ice handling
     yourConn.onicecandidate = function(event) {
+        console.log("Entering onicecandidate", event);
         if (event.candidate) {
             // comment in if you want to force relay, this demo currently requires
             // both callers to call if they want this.
```
Comment (dpwspoon) [inline gateway/webrtc/client.js]: remove this part of comment: // both callers to call if they want this. ?

File: gateway/webrtc/client.js (hunk 1 above)
Comment (DoruM) [reply gateway/webrtc/client.js]: No.

File: gateway/webrtc/client.js (hunk 2 above)
Comment (DoruM) [reply gateway/webrtc/client.js]: Ok

**Pull request merged**